VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,254 @@
+* Minimum self-bond amount: Minimum amount of Atoms the validator candidate need to have bonded at all time. If the validator's self-bonded stake falls below this limit, its entire staking pool will unbond.
+* Initial self-bond amount: Initial amount Atoms the validator candidate wants to self-bond
+
+Once a PubKey has declared candidacy, Atom holders can delegate atoms to it, effectively adding stake to this pool. The total stake of an address is the combination of Atoms bonded by users (called delegators) and Atoms self-bonded by the entity which designated itself.
+
+Out of all the candidates that signaled themselves, the 100 with the most stake are the ones who are designated as validators. If a validator’s total stake falls below the top 100 then that validator loses its validator privileges. Over time, the maximum number of validators will increase, according to a predefined schedule:
+
+* **Year 0:** 100
+* **Year 1:** 113
+* **Year 2:** 127
+* **Year 3:** 144
+* **Year 4:** 163
+* **Year 5:** 184
+* **Year 6:** 208
+* **Year 7:** 235
+* **Year 8:** 265
+* **Year 9:** 300
+* **Year 10:** 300
+
+### Is there a minimum amount of Atoms that must be staked to be a validator?
+
+There is no minimum. The top 100 validator candidates with the highest total stake (where total stake = self-bonded stake + delegators stake) are the validators. 
+
+### How will delegators choose their validators?
+
+Delegators are free to choose validators according to their own subjective criteria. This said, criteria anticipated to be important include:
+* **Amount of self-bonded Atoms:** Number of Atoms a validator self-bonded to its staking pool. A validator with higher amount of self-bonded Atoms has more skin in the game, making it more liable for its actions.
+* **Amount of delegated Atoms:** Total number of Atoms delegated to a validator. A high stake shows that the community trusts this validator, but it also means that this validator is a bigger target for hackers. Indeed, hackers are incentivized to hack bigger validators as they receive a reward proportionate to the stake of the validator they can prove to have compromised. Validators are expected to become less and less attractive as their amount of delegated Atoms grows.
+* **Track record:** Delegators will likely look at the track record of the validators they plan to delegate to. This includes seniority, past votes on proposals, historical average uptime and how often the node was compromised.
+
+Apart from these criteria that will be displayed in the Cosmos UI, there will be a possibility for validators to signal a website address to complete their resume. Validators will need to build reputation one way or another to attract delegators. For example, it would be a good practice for validators to have their setup audited by third parties. Note though, that the Tendermint Team will not approve or conduct any audit itself.
```

Comment:
"Tendermint Team" should maybe be "Tendermint and Cosmos Team"

Reply:
I thought there was no "Cosmos Team" ? Only Tendermint Team and ICF, Tendermint Team being contracted by ICF ?